assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -294,6 +294,9 @@ che.openshift.liveness.probe.delay=300
 che.openshift.liveness.probe.timeout=1
 che.openshift.workspaces.pvc.name=claim-che-workspace
 che.openshift.workspaces.pvc.quantity=10Gi
+che.openshift.workspace.cpu.limit=1
+# Override memory limit used for openshift workspaces. String, e.g. 1300Mi
+che.openshift.workspace.memory.limit=
```

Comment:
it would be nice if name of the property contains dementions like ```che.workspace.default_memory_mb```
Ex ```che.openshift.workspace.memory.limit_mb=```
Wdyt?